File: ExceptHandle/exceptHandle.cpp
#include <iostream>
using namespace std;
int main()
{
	int i = 10, j = 0, k;
	try
	{
		if (j == 0)
			throw j;
		else
		{
			k = i / j;
			cout << k;
		}
	}
	catch (int e)
	{
		if (e == 0)
			cout << "Divide by 0 error";
	}
	return 0;
}
/*Output
Divide by 0 error*/

File: constructor and exception handeling java/exceptHandle.java

class exceptHandle {
	public static void main(String[] args) {
		int i = 10, j = 0, k;
		try {
			k = i / j;
			System.out.println(k);
		} catch (Exception e) {
			System.out.println("Divide by 0 error");
		}
	}
}
/*
Output:
Divide by 0 error
*/